mifospay/src/main/java/org/mifos/mobilewallet/mifospay/standinginstruction/ui/NewSIActivity.kt

```diff
@@ -110,6 +121,56 @@ class NewSIActivity : BaseActivity(), StandingInstructionContract.NewSIView {
         ll_create_si.visibility = View.VISIBLE
     }
 
+    @OnClick(R.id.btn_scan_qr)
```

Comment:
We never use butterknife in kotlin, remove it. I admit the fact that this pattern is being followed in currently, this file will more probably need a rewrite to eliminate all the butterknife bindings.

Reply:
Yeah its true, i just followed the pattern of existing code.

Reply:
Building upon the existing caveats and error-prone code obliviously is never a good idea. When you know something better you must implement it instead of just connecting the dots. You can definitely follow the pattern if you're very sure of the fact it is good and safe to use practice.

Reply:
Thanks for correcting me😊
Will update the pr soon